core/src/main/java/io/keen/client/java/KeenClient.java

```diff
@@ -1549,13 +1565,19 @@ private void handleSuccess(KeenCallback callback) {
      * @param keenProperties  A Map that consists of key/value pairs to override default properties.
      *                        ex: "timestamp" -&gt; Calendar.getInstance()
      */
-    private void handleSuccess(KeenCallback callback, KeenProject project, String eventCollection, Map<String, Object> event,
-			Map<String, Object> keenProperties) {
-    	handleSuccess(callback);
+    private void handleSuccess(KeenCallback callback,
+                               KeenProject project,
+                               String eventCollection,
+                               Map<String, Object> event,
+                               Map<String, Object> keenProperties) {
+        handleSuccess(callback);
         if (callback != null) {
             try {
                 if(callback instanceof KeenDetailedCallback){
-                	((KeenDetailedCallback)callback).onSuccess(project, eventCollection, event, keenProperties);
+                    ((KeenDetailedCallback)callback).onSuccess(project,
+                                                               eventCollection,
+                                                               event,
+                                                               keenProperties);
                 }
             } catch (Exception userException) {
                 // Do nothing.
```

Comment:
This seems like a good question, especially because I think this is calling client code (the success callback). If that errors, the user should know about it.

Not required to fix in this PR, if you don't want to.